app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -226,7 +233,72 @@ enum SectionHeaderPlacement {
 const SECTION_HEADER_PLACEMENT: SectionHeaderPlacement = SectionHeaderPlacement::Bottom;
 
 
+
+// ========================
 // === Shape Definition ===
+// ========================
+
+
+// === Selection ===
+
+/// A shape of a selection box. It is used as a mask to show only specific parts of the selection
+/// layers. See [component_group](component_group#selection) module documentation to learn how this
+/// mask is used.
+///
+/// The selection `mask` is a rectangle with rounded corners. The final shape is an intersection of
+/// the `mask` and the `area` shape that covers the entire component list panel and has the same
+/// shape. This is needed to ensure that the selection box never leaves the borders of the
```

Comment:
I dont understand this sentence 

```
The final shape is an intersection of
  /// the `mask` and the `area` shape that covers the entire component list panel and has the same
  /// shape.
```

So, what has the same shape? The intersection and the component list?

Reply:
The whole section is rewritten for clarity.